crates/evm/src/evm/primitive_types.rs

```diff
@@ -12,6 +12,7 @@ pub(crate) struct BlockEnv {
     /// basefee is added in EIP1559 London upgrade
     pub(crate) basefee: u64,
     pub(crate) gas_limit: u64,
+    pub(crate) l1_fee_rate: u128,
```

Comment:
if you do this, method id is going to change and we have to do this carefully with a fork.

pls revert this and releated changes :)

Reply:
it might be a smart change tho. make an issue and label with requires fork etc.

Reply:
Reverted and created the issue #1266